src/components/CalendarMonth.jsx

```diff
@@ -25,6 +26,7 @@ const propTypes = forbidExtraProps({
   enableOutsideDays: PropTypes.bool,
   modifiers: PropTypes.object,
   orientation: ScrollableOrientationShape,
+  daySize: nonNegativeInteger,
```

Comment:
Tested with `daySize={-1}`, `nonNegativeInteger` without extra function invocation correctly prints to `console.error`.